8/readme.md

```diff
@@ -0,0 +1,41 @@
+# readme
+
+This example builds on top of solution 6 and demonstrates how to:
```

Comment:
This should probably get updated, but we can do it sometime later, when the whole example is complete.